scripts/run.sh

```diff
@@ -23,6 +23,8 @@ LOGIN="scala-steward"
 
 COMMON_ARGS=(
   --workspace "$STEWARD_DIR/workspace"
+  --repos-file "$STEWARD_DIR/repos.md"
+  --repos-default-conf "$STEWARD_DIR/default.scala-steward.conf"
```

Comment:
Could you please revert these two changes. I'm using this script to run my public Scala Steward instance. The `--repos-file` parameter is set later in this file and my instance will not use a default repo config.